.github/actions/latest-wrangler/README.md

```diff
@@ -0,0 +1,51 @@
+# Github package 'latest' tag wrangler for containers
+## Usage
+
+Plug in the necessary inputs to determine if the container being built should be tagged 'latest; at the package level, for example `dbt-redshift:latest`.
```

Comment:
Could you add some more about `latest` vs `minor_latest`? I want to make sure we capture why we need both tags and not just 1 (I could see this coming up as a question from a new hire or somebody outside of the team)

Reply:
TBH, I'm not 100% sure why we _need_ the `minor_latest` beyond the idea that we might have breaking changes between a `1.0.latest` and a `1.1.latest`?  @jerco would be a better person to answer that.

Reply:
Sorry I missed this comment a few days ago! Rationale is to provide consistency with how we're thinking about patch + minor upgrades elsewhere (e.g. in dbt Cloud).

There are two premises after v1.0:
- Use patches as soon as they're available
- Minor releases won't have breaking changes (we've committed to this!), but they will touch significantly more code than patch releases

As such, we should let users decide when they want to perform the minor version upgrade. Each minor version will be officially supported for 12 months after its initial release.

So I wouldn't be surprised if some users want to stick with `1.0.latest`, for a few weeks or a month, even after the `1.1.0` release. The inclusion of a `1.0.latest` tag will give them the opportunity to coordinate upgrades at a time of their choosing. During that period, if we have to cut a "security" patch for v1.0 (security bug, dependency/installation issue), they'll still get it automatically, as they should.